Content.Server/Chemistry/ReagentEffects/RemoveReagent.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Content.Server.Chemistry.EntitySystems;
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.FixedPoint;
+using JetBrains.Annotations;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Chemistry.ReagentEffects
+{
+    [UsedImplicitly]
+    public class RemoveReagent : ReagentEffect
```

Comment:
I'm not sure how often a reagent will try to remove just a single other reagent?  I imagine this would be much more useful if you could also target metabolism groups (e.g., remove poisons)?

Reply:
This would be broadly useful for chems like Charcoal if it could remove poisons, yea

Reply:
no reason it can't do both